src/prefect/settings.py

```diff
@@ -828,12 +828,12 @@ def default_cloud_ui_url(settings, value):
 
 PREFECT_LOGGING_INTERNAL_LEVEL = Setting(
     str,
-    default="ERROR",
+    default="WARNING",
```

Comment:
I feel like it is okay to move this to `WARNING` so we can emit things we care about from the service. Alternatively could write to `sys.stderr` directly? Or can investigate taking a totally different approach to this pr.

Reply:
I'd rather keep this at `ERROR` and use a different logger for the new message. The `GlobalEventLoopThread | prefect._internal.concurrency` in the logs you posted in the PR description seems like unnecessary information.

Reply:
that's fair! let me see what I can do

Reply:
+1 exposing that `_internal` package feels icky

Reply:
Okay I've updated the service to have it's own logger. I am somewhat scared about this just based on this file: https://github.com/PrefectHQ/prefect/blob/d2de43322241d1e5668684b6047e76ee4ebd7830/src/prefect/_internal/_logging.py#L1-L26

but now it looks like this:
```
13:36:30.706 | INFO    | Task run 'add_one-b72' - Finished in state Completed()
13:36:30.706 | INFO    | Task run 'add_one-df7' - Finished in state Completed()
13:36:31.154 | INFO    | Flow run 'ninja-weasel' - Finished in state Completed()
13:36:31.156 | WARNING | EventsWorker - Still processing items: 941 items remaining...
13:36:34.159 | WARNING | EventsWorker - Still processing items: 715 items remaining...
13:36:37.163 | WARNING | EventsWorker - Still processing items: 482 items remaining...
13:36:40.174 | WARNING | EventsWorker - Still processing items: 248 items remaining...
13:36:43.181 | WARNING | EventsWorker - Still processing items: 18 items remaining...
```

Reply:
I could add another implementation of `SafeLogger` that doesn't depend on `PREFECT_LOGGING_INTERNAL_LEVEL`?

Reply:
re-opened here: https://github.com/PrefectHQ/prefect/pull/14964